country.php

```diff
@@ -52,17 +52,57 @@
 			  require("nav.php");
 			?>
 			
-			<!-- banner class to establish what the site is about and what further content may entail (will be an image in later stages)-->
+			<!-- image banner class to establish what the site is about and what further content may entail-->
 			<!-- banner uses the parallax scrolling effect (for background image) -->
-			<center><div class="banner" style="background-image: url('images/country-banner.jpg'); background-size: 100vw 100vh; background-attachment: fixed;">
-				<h1 style="padding-top: 18vw; font-size: 4.8vw; color: white;">COUNTRY</h1>
-				<h2 style="margin-top: -1.5vw; font-size: 1.75vw;">All Country And Acoustic Songs</h2>
-				<img style="width: 100%; max-width: 20px;" src="images/clock.png" alt="Clock Icon">
-				<h2 style="margin-top: 0; padding-left: 5px; display: inline-block">01:32:24</h2>
-			</div></center>
+			<div class="center">
+				<div class="banner" style="background-image: url('images/country-banner.jpg'); background-size: 100vw 100vh; background-attachment: fixed;">
+				<div class="text-overlay">
```

Comment:
#used text-overlay div to position the text in my banner to be directly in the center rather than using padding to estimate